readme.md

```diff
@@ -60,9 +60,36 @@ InnerLoadingSpinerComponent.propTypes = {
 + export const LoadingSpinnerComponent = promiseTrackerHoc(InnerLoadingSpinerComponent);
 ```
 
+```diff
+import React from 'react';
++ import { LoadingSpinnerComponent} from './loadingSpinner';
+
+export const AppComponent = (props) => (
+  <div>
+    <h1>Hello App!</h1>
++   <LoadingSpinnerComponent />
+  </div>
+);
+```
+
+## Sample with areas:
+
```

Comment:
Explain here what areas and usage sample

Reply:
Ok, thank you

Reply:
Done